shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/model/ObjectCollectionBuilder.kt

```diff
@@ -93,11 +93,11 @@ class ObjectCollectionBuilder {
 
     class LineBuilder : ObjectBuilder<Line> {
         var id = uuid()
-        var color = ""
+        var color = "FFFFFF"
```

Comment:
Got an error about the default hex code not being 6 characters. We are already defaulting to these on the RouteBuilder.